metricflow/plan_conversion/dataflow_to_sql.py

```diff
@@ -1390,6 +1361,21 @@ def visit_semi_additive_join_node(self, node: SemiAdditiveJoinNode) -> SqlDataSe
             ),
         )
 
+    def _choose_instance_for_time_spine_join(
```

Comment:
This helper function is only used once at this point, but it will be used again for the `JoinToTimeSpineNode` farther up the stack.